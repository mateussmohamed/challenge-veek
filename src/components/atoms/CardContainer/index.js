import styled from 'styled-components'

const CardContainer = styled.article`
  display: flex;
  flex-direction: column;
  padding: 0;
  margin: 0;
  width: 100%;
  border: 1px solid #dfdfdf;
  margin-bottom: 45px;
  background-color: #fff;
`

export default CardContainer
