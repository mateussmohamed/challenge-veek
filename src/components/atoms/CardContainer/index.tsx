import styled from 'styled-components'

export const CardContainer = styled.article`
  display: flex;
  flex-direction: column;
  padding: 0;
  margin: 0;
  width: 100%;
  border: 1px solid #dfdfdf;
  margin-bottom: 45px;
  background-color: #fff;
`
