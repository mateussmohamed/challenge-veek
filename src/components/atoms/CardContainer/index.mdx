---
name: CardContainer
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import CardContainer from './index'

# CardContainer
<PropsTable of={CardContainer} />

<Playground>
  <CardContainer />
</Playground>
