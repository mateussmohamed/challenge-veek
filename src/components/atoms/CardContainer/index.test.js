import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import CardContainer from './index'

describe('[<CardContainer />]', () => {
  const wrap = (props = {}) => shallow(<CardContainer {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am a container!' })

    expect(wrapper).toMatchSnapshot()
  })
})
