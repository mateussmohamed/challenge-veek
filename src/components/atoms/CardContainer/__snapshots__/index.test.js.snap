// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<CardContainer />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "CardContainer-sc-1wp1g4b-0",
        "isStatic": true,
        "rules": Array [
          "display:flex;flex-direction:column;padding:0;margin:0;width:100%;border:1px solid #dfdfdf;margin-bottom:45px;background-color:#fff;",
        ],
      },
      "displayName": "CardContainer",
      "render": [Function],
      "styledComponentId": "CardContainer-sc-1wp1g4b-0",
      "target": "article",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  I am a container!
</StyledComponent>
`;
